.github/reviewers.yaml

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  defaults:
+    - team:maintainers
+    - team:contour-reviewers
+
+options:
+  ignore_draft: true
+  number_of_reviewers: 2
```

Comment:
Is there any way to ensure that at least one maintainer is selected?

Reply:
ah yeah there might be a way to do so with the "groups" feature, but it might be simpler to keep the CODEOWNERS and just specify reviewers with this action

Reply:
Circling back here, I'm in favor of keeping CODEOWNERS for existing maintainers, and using this workflow for assigning non-maintainer reviewers.